### PR TITLE
Only import zmesh if called

### PR DESCRIPTION
**Pull request by j6k4m8**
You don't need zmesh installed unless you're meshing something.

### CODE REVIEW

File: intern/service/mesh/service.py [hunk 1]
```diff
@@ -13,21 +13,23 @@
...
-    nm = 1,
-    um = 1000,
-    mm = 1000000,
-    cm = 10000000,
-    nanometers = 1, 
-    micrometers = 1000, 
-    millimeters = 1000000, 
-    centimeters = 10000000,
+
+    nm = (1,)
```
Comment (movestill) [inline intern/service/mesh/service.py:24]: Why are these tuples, now?

Comment (j6k4m8) [reply intern/service/mesh/service.py:24]: oh shoot i must have accidentally linted this full file. let me fix that...

Comment (j6k4m8) [reply intern/service/mesh/service.py:24]: although — it looks like this comma-notation in the enum means that the interpreter was reading these types as `Tuple[int]`s anyway. The linter just added parentheses for clarity. I _think_ that's not what we wanted, but check this out:

```python
from enum import IntEnum

class X(IntEnum):
    a = (1,)
    b = (2,)

X.a == 1 # True
```

`IntEnum` does some weird stuff here (details on SO [here](https://stackoverflow.com/a/26125425/979255)); i have a feeling we've been using tuples all along...

**Comment by j6k4m8** [comment]
Updated the docstring, fixed tuple→int, and moved zmesh import to inside the invocation so you don't need it to install intern. Ready for your thoughts :)

**Pull request merged**